languages/scala/exercises/concept/operators/.docs/instructions.md

```diff
@@ -0,0 +1,60 @@
+In this exercise you're going to write a calculator for rectangles.
+
+Given is a case class `Rectangle`. A rectangle is defined by its width and its height:
+
+```scala
+case class Rectangle(width: Int, height: Int)
+```
+
+The calculator will allow to sum up two rectangles and to increase a rectangle by multiplying it with a number.
+
+## 1. Create a plus operator for rectangles
```

Comment:
This heading is a bit technical. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd):

> The heading should describe what to implement, not how to implement it 

Maybe something like:

```suggestion
## 1. Allow rectangles of the same height to be added
```

Or something like that.

Reply:
Thanks, good general hint.